charts/helm-broker/values.yaml

```diff
@@ -52,10 +52,10 @@ global:
     pullPolicy: IfNotPresent
   # paths to docker images
   helm_broker:
-    dir: /pr/
+    dir: /
     version: PR-153
   helm_controller:
-    dir: /pr/
+    dir: /
```

Comment:
this changes how the image gets rendered:
```
eu.gcr.io/kyma-project/pr/helm-controller:PR-159  # original
eu.gcr.io/kyma-project/helm-broker:PR-153         # proposed
```
the original doesn't seem to exist
```
$ docker pull eu.gcr.io/kyma-project/pr/helm-controller:PR-159
Error response from daemon: manifest for eu.gcr.io/kyma-project/pr/helm-controller:PR-159 not found: manifest unknown: Failed to fetch "PR-159" from request "/v2/kyma-project/pr/helm-controller/manifests/PR-159".
```

I have checked that without `/pr/`, however, does exist
```
$ docker pull eu.gcr.io/kyma-project/helm-broker:PR-153
PR-153: Pulling from kyma-project/helm-broker
Digest: sha256:20924cc4a7b5173d12efdab3d10c9ad3d05b4398e934d784e54a71f41021ab55
Status: Image is up to date for eu.gcr.io/kyma-project/helm-broker:PR-153
eu.gcr.io/kyma-project/helm-broker:PR-153
```
Perhaps the `/pr/` in the image path was extra?